airbyte-webapp/src/components/ui/TagInput/TagInput.module.scss

```
@@ -0,0 +1,9 @@
+@use "../../../scss/colors";
+@use "../../../scss/variables";
```

As it turned out we can import scss without using the relative path
```suggestion
@use "scss/colors";
@use "scss/variables";
```